### DISCUSSION TITLE
Struggling to figure out how to make a Chord diagram

### DISCUSSION THREAD

**Discussion by janzheng** [observablehq/plot#2259] [Q&A]
I've been trying to figure out how to create this D3 Chord diagram in Observable Plot, but I don't think that's possible?

https://observablehq.com/@d3/chord-dependency-diagram/2?collection=@d3/d3-chord

Any help / examples appreciated! I'm just using the d3 library now though so no pressure, but I couldn't find examples, and struggling to make Vector/Arrow marks work.

**Comment by Fil** [comment]
Correct: there is no equivalent to D3's chord diagram in Observable Plot. (See also https://github.com/observablehq/plot/issues/133)

Please give more specifics about the issues you had using the arrow and vector marks.
